src/components/atoms/MenuLink/styles.tsx

```diff
@@ -15,10 +15,13 @@ export const StyleNavMenuLink = styled.div`
     }
 `;
 
+export const StyleLink = styled(Link)`
+    text-decoration: none;  
+`;
+
 export const StyleLinkName = styled.span`
     color: ${colors.gray.darker};
     font-size: ${fsize.bigger};
     margin: auto;
```

Comment:
Are you using margin:auto to center something?

Reply:
I was using it to make sure that the link name would be centered within the StyleNavMenuLink div as well. If it's redundant, I can delete it.

Reply:
In that case, I think that a safer approach will be `margin: 0 auto;` or `text-align:center`

Reply:
Okay, I went with text-align:center. Look fine now.